pf/internal/pfutils/proposed_new_test.go

```diff
@@ -28,12 +30,16 @@ import (
 )
 
 type testcase struct {
-	Schema Schema
+	Schema runtypes.Schema
 	Prior  valueBuilder
 	Config valueBuilder
 	Want   valueBuilder
 }
 
+type runtiemSchemaAdapter struct{ Schema }
+
+func (runtiemSchemaAdapter) Shim() shim.SchemaMap { panic("UNIMPLIMENTED") }
```

Comment:
```suggestion
func (runtimeSchemaAdapter) Shim() shim.SchemaMap { panic("UNIMPLEMENTED") }
```

Note also typo in unimplemented